app/models/form/lettings/questions/person_partner.rb

```diff
@@ -0,0 +1,30 @@
+class Form::Lettings::Questions::PersonPartner < ::Form::Question
+  def initialize(id, hsh, page, person_index:)
+    super(id, hsh, page)
+    @id = "relat#{person_index}"
+    @type = "radio"
+    @check_answers_card_number = person_index
+    @answer_options = answer_options
+    @person_index = person_index
+    @question_number = question_number
+  end
+
+  def answer_options
+    {
+      "P" => { "value" => "Yes" },
+      "X" => { "value" => "No" },
+      "R" => { "value" => "Tenant prefers not to say" },
+    }
+  end
+
+  def question_number
+    base_question_number = case form.start_date.year
+                           when 2023
```

Comment:
Is this needed?

Reply:
hmm, no, I thought I'd leave it until we get the question numbers